app/retail/emails.py

```diff
@@ -1250,6 +1264,13 @@ def comment(request):
     return HttpResponse(response_html)
 
 
+@staff_member_required
+def mention(request):
```

Comment:
is this for testing? Is there a URL that points to this view?

Reply:
No, but I figured that this might be a function that can be called by a staff member to manually trigger the email, since other kinds of notifications also have a similar method (look above and below).

Reply:
I see - normally we hook these views into the router behind our admin endpoint like this: https://github.com/gitcoinco/web/blob/master/app/app/urls.py#L486
That way our design team can check out what the email looks like - can you go ahead and hook this view into the router? thanks!

Reply:
Done!